xbmc/Application.cpp

```diff
@@ -1101,6 +1101,8 @@ bool CApplication::Initialize()
       }
     }
 
+    CServiceBroker::GetServiceAddons().Start();
```

Comment:
?? This is before login. The correct one is on line 1175

Reply:
1175 is also called before login if the login screen on startup is enabled. After combining the Start methods as you suggest, I can just remove this line; it will be called when the profile logs in.